apache/superset
# fix(dashboard): Remove bar at bottom of dashboard edit sidebar

### SUMMARY

This PR removes a bar that was appearing at the bottom of the dashboard edit sidebar.  It does some misc cleanup along the way:
- Removes `builder-sidepane.less`, which contains a few actively-used styles but is mostly unused styles, and moves the actively-used styles into the relevant component files
- Replaces use of `react-sticky` with  CSS `position: sticky` (which seems to now be [supported](https://caniuse.com/css-sticky) for all of the browsers we're targeting) for the sidebar
- Removes various styles and props that are now unnecessary because of above changes

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


Before:
<img width="900" alt="Screen Shot 2022-10-13 at 8 21 07 PM" src="https://user-images.githubusercontent.com/13007381/195747350-71070bc3-6da5-4a3b-8d72-6a7f7fd81750.png">

After:
![localhost_9000_superset_dashboard_10__native_filters_key=7nf2YdCXSNoo89wKJp6bQ0tjrlAX2AYcEkPxtZDcKB7rOM5_3bFogSmpuKjZxj1o (1)](https://user-images.githubusercontent.com/13007381/195747009-7742d050-522f-4c4d-bb38-109bfe57c13a.png)

### TESTING INSTRUCTIONS

- Check that the bar is gone
- Check that the Edit sidebar otherwise looks identical
- Check that the Edit sidebar looks identical in standalone mode (add `standalone=1` to the end of the query string)

### ADDITIONAL INFORMATION
